ui/raidboss/data/05-shb/raid/e5s.js

```diff
@@ -175,10 +236,15 @@
       regexFr: Regexes.startsUsing({ id: '4BAD', source: 'Ramuh', capture: false }),
       regexJa: Regexes.startsUsing({ id: '4BAD', source: 'ラムウ', capture: false }),
       regexKo: Regexes.startsUsing({ id: '4BAD', source: '라무', capture: false }),
-      infoText: {
-        en: 'Be in your position',
-        fr: 'Soyez en place',
-        ko: '자기 위치에 있기',
+      infoText: function(data) {
+        if (!data.furysBoltActive) {
+          return {
+            en: 'Short Knockback',
+          };
+        }
+        return {
+          en: 'Big Knockback',
+        };
```

Comment:
`E5S Centaur\'s Charge` is always Big Knockback.
It happens only after fury's fourteen.
I think you were thinking about the add pushing party members from the center to sides, which is completely not dealt with e5s trigger now.
Think of that, tank moves Ramuh during first pushing.

Reply:
Understood; yea, I had those confused then.